src/state/actions/reduxTokenAuthConfig.js

```diff
@@ -0,0 +1,28 @@
+import { generateAuthActions } from "redux-token-auth";
+
+let authUrl = "http://localhost:3000/api/auth";
+
+const config = {
+  authUrl,
+  userAttributes: {
+    firstName: "first_name",
+    lastName: "last_name",
+    email: "email"
+  },
+  userRegistrationAttributes: {
+    firstName: "first_name",
+    lastName: "last_name",
+    email: "email",
+    password: "password",
+    password_confirmation: "password_confirmation"
```

Comment:
we only need to send one password to the API

Reply:
fixed